implementation-status.md

```diff
@@ -32,13 +30,18 @@ Tip: Chrome channel releases are tracked at [https://googlechromelabs.github.io/
 * [Issue 854935](http://crbug.com/854935): Closed captions in Picture-in-Picture are not supported.
 
 # Microsoft Edge
-https://dev.windows.com/en-us/microsoft-edge/platform/status/pictureinpicture
+Under consideration.
+https://developer.microsoft.com/en-us/microsoft-edge/platform/status/pictureinpicture/
 
 # Safari
+No public signals. 
```

Comment:
My understanding is that Safari is supportive and has their own private API.

Reply:
Oh, do they expose some `webkitThing.` thing?

Reply:
Yeah, I think it's something like `video.webkitSetPresentationMode("picture-in-picture")`.

Reply:
Neat, I had no idea.

Reply:
From https://developers.google.com/web/updates/2018/10/watch-video-using-picture-in-picture#background,

> In September 2016, Safari added Picture-in-Picture support through a WebKit API in macOS Sierra. Six months later, Chrome automatically played Picture-in-Picture video on mobile with the release of Android O using a native Android API. Six months later, we announced our intent to build and standardize a Web API, feature compatible with Safari’s, that would allow web developers to create and control the full experience around Picture-in-Picture. And here we are!

Reply:
Moreover, they're accepting patches for review: https://bugs.webkit.org/show_bug.cgi?id=189848#c8

Reply:
Ok, nice. This is encouraging!

Reply:
FYI I've pushed https://github.com/WICG/picture-in-picture/pull/154